client/src/Movies/MovieCreate.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import axios from 'axios';
+
+class MovieCreate extends React.Component{
+    constructor(props){
+        super(props);
+        this.state={
+            title:'',
+            director:'',
+            metascore:'',
+            stars:''
+        }
+    }
+    onInputChange=event=>{
+        this.setState({[event.target.name]:event.target.value});
+    }
+    submitMovie=()=>{
+        const newMovieObj={
+            title:this.state.title,
```

Comment:
The way the newMovieObj is set up now, it is not passing the constraints of the post endpoint. Look at the server.js file again and the logic inside of the post endpoint.